modin/logging/config.py

```diff
@@ -144,8 +144,10 @@ def memory_thread(logger, sleep_time):
         The interval at which to profile system memory.
     """
     while True:
+        rss_mem = bytes_int_to_str(psutil.Process().memory_info().rss / 1000000)
```

Comment:
```suggestion
        rss_mem = bytes_int_to_str(psutil.Process().memory_info().rss / 1_000_000)
```

Reply:
I think this got updated, so we use `bytes_int_to_str` instead of dividing by any number here.